notebooks/official/model_monitoring/model_monitoring.ipynb

```diff
@@ -336,13 +336,14 @@
       "source": [
```

Comment:
<h2>Deploy your endpoint -&gt; Deploy your Model resource to an Endpoint resource</h2><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/859/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>fixed</p>